modules/install/pages/thp-disable.adoc

```diff
@@ -3,18 +3,24 @@
 :tabs:
 
 [abstract]
+--
 {description}
```

Comment:
I believe the current description can remain the same, especially keeping the exsiting point of "THP must be disabled in order for Couchbase Server to function correctly on Linux.".

We could add to the abstract that having THP enabled can worsen performance and possibly lead to an OOM kill. However, these points I believe should remain concise within the abstract.